tests/__tests__/sitka_module.test.ts

```diff
@@ -86,13 +89,39 @@ describe("SitkaModule", () => {
     expect(currentState).toEqual(defaultTextModuleState)
   })
 
-//   // SUBSCRIPTION
-//   test('subscriptions are created/provided with provideSubscriptions & createSubscription', () => {
-//   })
+  // SUBSCRIPTION
+  test('subscriptions are created/provided with provideSubscriptions & createSubscription', () => {
+    const { sitka, store } = createSitkaAndStore()
+    const { text: textModule, color: colorModule } = sitka.getModules()
+    // Validates that we start with default state
+    const startingEditsState = textModule.getStateTestDelegate(store.getState()).numberOfEdits
```

Comment:
no need to use the delegate to inspect store state, use store state. getStateTestDelegate should ONLY be used to test getState.

Reply:
removed except in the first instance where needed